src/Extensions/TimedScopes/Internal/ActivityObserversIntializer.cs

```diff
@@ -14,6 +14,21 @@ namespace Microsoft.Omex.Extensions.TimedScopes
 {
 	internal sealed class ActivityObserversIntializer : IHostedService, IObserver<DiagnosticListener>, IObserver<KeyValuePair<string, object>>
 	{
+		private const string ActivityStartEnding = ".Start";
+
+		private const string ActivityStopEnding = ".Stop";
+
+		private static readonly string[] s_eventEndMarkersToListen = new[] {
+			ActivityStartEnding,
+			ActivityStopEnding,
+			// we need to listen for "Microsoft.AspNetCore.Hosting.HttpRequestIn" event in order singnal Kestrel to create Activity for incoming http request
+			// searching of for RequestIn in case any other requests follow the same pattern
+			"RequestIn",
+			// we need to listen for "System.Net.Http.HttpRequestOut" to create activity for outgoing http requests
+			// searching only for RequestOut in case any other requests follow the same pattern
+			"RequestOut",
```

Comment:
Why not make these two string consts also?

Reply:
Because we are not reusing it anywhere

Reply:
Sure. It just looked inconsistent :).